### PR TITLE
add a safety check for getproperty

### PR DESCRIPTION
**Pull request by taooceros**
add a safety check for https://github.com/Flow-Launcher/Flow.Launcher/issues/2395#issuecomment-1803229966

This seems to be the only place that we didn't check whether property exists.

### CODE REVIEW

File: Plugins/Flow.Launcher.Plugin.BrowserBookmark/ChromiumBookmarkLoader.cs [hunk 1]
```diff
@@ -2,12 +2,14 @@
 using System.Collections.Generic;
 using System.IO;
 using System.Text.Json;
+using Flow.Launcher.Infrastructure.Logger;
```
Comment (jjw24) [inline Plugins/Flow.Launcher.Plugin.BrowserBookmark/ChromiumBookmarkLoader.cs:5]: Infrastructure Logger, What's this for?

Comment (taooceros) [reply Plugins/Flow.Launcher.Plugin.BrowserBookmark/ChromiumBookmarkLoader.cs:5]: oh mb forget about this is for plugin. Though I think somewhere else is still using Infra? I will check soon

Comment (taooceros) [reply Plugins/Flow.Launcher.Plugin.BrowserBookmark/ChromiumBookmarkLoader.cs:5]: Yeah we ref Infra for Enum displaying...Unless we want to get rid of that it seems to ok to ref the static Log?

Comment (jjw24) [reply Plugins/Flow.Launcher.Plugin.BrowserBookmark/ChromiumBookmarkLoader.cs:5]: hmm actually used for calling logging, when we redo logging we can move it to Plugin project so there is less/no dependency on the Infrastructure project.

**Review by jjw24 — APPROVED** [review]

**Pull request merged**